src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -125,9 +159,42 @@ public static void LogMetric(this ILogger logger, string name, double value, Dic
 
             context = context ?? new Dictionary<string, object>();
 
-            logger.LogInformation(
-                $"{MessagePrefixes.Metric} {{{ContextProperties.MetricTracking.MetricName}}}: {{{ContextProperties.MetricTracking.MetricValue}}} (Context: {{@{ContextProperties.EventTracking.EventContext}}})",
-                name, value, context);
+            logger.LogInformation(MetricFormat, name, value, context);
+        }
+
+        /// <summary>
+        ///     Logs an event related to an security activity (i.e. input validation, authentication, authorization...).
+        /// </summary>
+        /// <param name="logger">The logger to use.</param>
+        /// <param name="categoryName">The name of the security category.</param>
+        /// <param name="message">The user message written with in the <paramref name="categoryName"/>.</param>
+        /// <param name="arguments">The arguments formatted with the <paramref name="message"/>.</param>
+        public static void LogSecurityEvent(this ILogger logger, string categoryName, string message, params object[] arguments)
```

Comment:
In my opinion, we should guide our consumers to write good traces so we should enforce this via our parameters.

Now we are just mapping it directly on what .NET enforces us to give

Reply:
Think what we need is:
- Security event name
- Details about the event

Something like this? But it's hard to come up with this without a concrete case

Reply:
For example something I think about during input validation:

```csharp
logger.LogError("{SecurityCategory} - input is invalid person's age, > {MaxAge}", "Input validation", 130);
```

Reply:
That's also one of the reasons I initially said we could use an enricher, bc I can think of different type of scenario's with different kind of critical levels, with different kind of arguments to send,...

Reply:
Not sure why that's a security event imo but I see your point.

In that case we have a security category, message and parameters, thinkthat should cut it. You cannot use an enricher for this kind of stuff because enrichers have no knowledge of the scenario, they just enrich based on existing messages 🤷‍♂

Reply:
It's about diagnostics you know, if you see for example an input validation mutiple times, see a pattern... you can figure out more easily malicious activity. And with adding an extra security property, makes the filtering more easily.